artifact-new.md

```diff
@@ -0,0 +1,62 @@
+Steps to Create an Artifact
+The process of adding any variety of content to PECE begins by logging in with your screen name and password and clicking “Create Content” in the top right corner.
+
+![](media/PECE new artifact create content screenshot.png)
+
+You should now see the "Add content" page.
+
+![](media/PECE-new-artifact-add-content-screenshot.png)
+
+On this page you will find a grid of options for creating different forms of new content followed by their brief descriptions. “Artifacts” are one type of content. An artifact is the basic working unit of PECE; once, created, artifacts are then used in photo essays (for image artifacts only), timelines, or PECE Essays (both of which accept images, video, documents, and any other type of artifact)
+
+You can add many types of artifacts, including audio files, video, images, pdf, text, and websites. Within the "Archive" section of content types, you will see all the artifact options.
+
+Next, click the type of artifact you want to create.
+
+![](media/PECE-new-artifact-screenshot-image.png)
+
+
+You should now have access to all of the fields you need to fill out in order to create a new image artifact. First, type the title of your image in the “Title” pane.
+
+Next, upload your artifact by clicking the "Choose file" button and selecting the name of the file you would like to upload. 
+
+![](media/PECE-new-artifact-title-screenshot.png)
+
+
+Click “Open” to upload the file into PECE. Depending on the size of the file it may take a few seconds or minutes to upload; a small icon for the file will appear once uploading is complete.
+Add Alt Text as desired.
+
+Next, scroll down to the metadata fields. The menu bar on the left side will automatically be open to the "Permission, Authorship, and Groups" section. Your screen name should already appear in the first contributor field. To add the names of any other project members who you would like to grant access to editing the artifact, type the first few letters of their name into the other “contributor” field. To add more than one additional contributor, click the "add another item" button and enter their name in the new field.
+
+
+Next, scroll down to the "View Permission and Groups" section. The button switch will automatically set viewing permissions to "Everyone can view this content," making the artifact visible to all internet users. To change viewing access, click the button switch. A new field "People with view access" will pop up. To allow specific people viewing access, type the first few letters of their name and select their username.
```

Comment:
```suggestion
Next, scroll down to the "View Permission and Groups" section. The button switch will automatically set viewing permissions to "Everyone can view this content," making the artifact visible to all internet users. If you want to restrict access to the content, click the button switch to turn this off.

If you choose to leave the toggle on, you can still associate the content with a particular group. To do so, complete the "Group association and view access." You can select a group with which to associate the content by typing its name into the field.

If you choose to restrict access to the content, you can designate specific people or groups that will have view access. To allow specific people viewing access, in the field, "People with view access" type the first few letters of their name and select their username. Add groups whose members should have view access to this memo by typing the group name into the "Group association and view access" field. **Note that if you give a group view access, it will also associate the content with that group. This means the content will appear on the group's page.**
```